app/models/changeset/pull_request.rb

```diff
@@ -129,9 +129,17 @@ def message
 
   private
 
+  def section_content(section_title, text)
+    desired_header_regexp = "^(?:\\s*#+\\s*#{section_title}.*|\\s*#{section_title}.*\\n\\s*(?:-{2,}|={2,}))\\n"
+    content_regexp = '([\W\w]*?)'
```

Comment:
how is that different from `.*?` ?

Reply:
`.` matches everything but a newline, while `\W` will match the newline.